Assets/MixedRealityToolkit-SDK/Features/Input/FocusProvider.cs

```diff
@@ -298,38 +292,52 @@ public bool TryGetPointingSource(BaseInputEventData eventData, out IMixedReality
         /// <inheritdoc />
         public GameObject GetFocusedObject(IMixedRealityPointer pointingSource)
         {
+            if (pointingSource == null)
```

Comment:
I think this should be below the `OverrideFocusedObject` check. That should override everything.